float32[] vfPathPositionX
float32[] vfPathPositionY
float32[] vfConstraintPositionX
float32[] vfConstraintPositionY
---
bool boSuccess